tools/dailyjobs_query.py

```diff
@@ -0,0 +1,126 @@
+import MySQLdb
+import datetime
+from argparse import ArgumentParser
+import sys
+
+revisions_dict = {}
+database = MySQLdb.connect(host="localhost",
+                         user="root",
+                         passwd="root",
+                         db="ouija")
+cur = database.cursor()
+
+branches = ["mozilla-inbound", "fx-team"]
+platforms = ["linux", "osx", "win", "android"]
+
+
+def parse_args(argv=None):
+    parser = ArgumentParser()
+    parser.add_argument("-s", "--start_date",
```

Comment:
Please use --start-date rather than --start_date, and --end-date rather than --end_date.